src/backend/server.js

```diff
@@ -265,7 +265,10 @@ app.post('/logout',
 )
 
 app.get('/admin/events/create', isAuthenticated, wrap(async (req, res) => {
-  res.send(createEventPage({is_public: false}));
+  //res.send(createEventPage({is_public: false}));
+  const t = fs.readFileSync(templateDir + '/create_event.hbs', {encoding: 'utf-8'});
```

Comment:
Wouldn't use readFileSync here.  It blocks the main event thread, which causes all requests to wait for this operation to complete.  Instead, use readFile with a callback or, if you want to be consistent with how we do things in the rest of Ground Control, use a promise and then call await on the function.  I can explain further what that means if that doesn't make sense.